data/ap-to-csv

```diff
@@ -93,12 +96,6 @@ function doResultsFile (file, done) {
 }
 
 function processReportingUnit (ru) {
-  // avoid duplicate states
```

Comment:
This line was causing the map to show the wrong result, since it would read a state-level senate result, for instance, and assume that was the presidential result.